src/format.rs

```diff
@@ -524,6 +524,10 @@ impl Documentable for &UntypedClauseGuard {
                 left.as_ref().to_doc().append(" > ").append(right.as_ref())
             }
 
+            ClauseGuard::GtFloat { left, right, .. } => {
+                left.as_ref().to_doc().append(" >. ").append(right.as_ref())
```

Comment:
In Erlang the operator is `>`, so we'll want to render that rather than `>.` here.

Reply:
Well spotted, thank you. Have fixed this below.